jenkins/databricks/build.sh

```diff
@@ -15,437 +15,292 @@
 # limitations under the License.
 #
 
-set -ex
+set -e
 
 ## Environments SPARKSRCTGZ, BASE_SPARK_VERSION, BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS, MVN_OPT
-## can be overwritten by shell variables, e.g. "BASE_SPARK_VERSION=3.1.2 MVN_OPT=-DskipTests bash build.sh"
-
-SPARKSRCTGZ=${SPARKSRCTGZ:-''}
-# version of Apache Spark we are building against
-BASE_SPARK_VERSION=${BASE_SPARK_VERSION:-'3.1.2'}
-BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS=${BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS:-$BASE_SPARK_VERSION}
-## '-Pfoo=1,-Dbar=2,...' to '-Pfoo=1 -Dbar=2 ...'
-MVN_OPT=${MVN_OPT//','/' '}
-
```

Comment:
nit: avoid white space changes

Reply:
Sorry for the inconvenience, those lines were indented because they moved inside function "`initialize`"